master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -510,7 +510,9 @@ private[celeborn] class Master(
               requestSlots.shouldReplicate,
               diskReserveSize,
               slotsAssignLoadAwareDiskGroupNum,
-              slotsAssignLoadAwareDiskGroupGradient)
+              slotsAssignLoadAwareDiskGroupGradient,
+              conf.slotsAssignLoadAwareFlushTimeWeight,
```

Comment:
According to our previous code, I think adding a variable or changing the existing code style will be better.

Reply:
done